app/src/main/java/io/neurolab/tools/Utils.java

```diff
@@ -0,0 +1,35 @@
+package io.neurolab.tools;
+
+import java.io.BufferedInputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+
+public class Utils {
+
+    public static int countLines(String filename) throws Exception {
+        return countLines(new File(filename));
+    }
+
+    public static int countLines(File file) throws Exception {
+
+        try (InputStream input = new BufferedInputStream(new FileInputStream(file))) {
+            byte[] c = new byte[1024];
+            int count = 0;
+            int readChars = 0;
+            boolean empty = true;
+            while ((readChars = input.read(c)) != -1) {
+                empty = false;
+                for (int i = 0; i < readChars; ++i) {
+                    if (c[i] == '\n') {
+                        ++count;
+                    }
+                }
+            }
+            return (count == 0 && !empty) ? 1 : count;
+        } catch (Exception e) {
+            e.printStackTrace();
+            throw new Exception(e.getMessage());
```

Comment:
Better just throw the exception itself. This will limit the stacktrace

Reply:
Still same. Please don't resolve issues yourself. GitHub will automatically outdate them once you make changes

Reply:
Okay will update.